Add doc for widget examples input (#454)
huggingface/huggingface_hub: docs/hub/model-repos.md

- text: "Jens Peter Hansen kommer fra Danmark"
```

> It is also possible to specify multiple example inputs & their titles that will be used in the examples picker drop-down of the widget.

Reviewer: Maybe include a screenshot of picking Sentiment Analysis from the dropdown menu for T0pp, and then match the example YAML snippet below to clearly link the two?

```suggestion
If you have more than one example input, you can name each example input with `example_title`. Users can now select a specific example using the `example_title` from the dropdown menu of the widget.
```

```yaml
widget:
- text: "Paris is the <mask> of France."
  example_title: "Geography"
- text: "The goal of life is <mask>."
  example_title: "Philosophy"
```

> Moreover, you can specify non-text example inputs in the model card metadata. Find [here](https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/input-examples.md) an exhaustive list of input examples formats for all widget types. For vision & audio widget types, you can provide example inputs under key `src` rather than `text`. For example, allow users to choose from two sample audio files for automatic speech recognition tasks by:

Reviewer: ```suggestion
Moreover, you can specify non-text example inputs in the model card metadata. Refer [here](https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/input-examples.md) for a complete list of input example formats for all widget types. For vision & audio widget types, provide example inputs with `src` rather than `text`. 

For example, allow users to choose from two sample audio files for automatic speech recognition tasks by:
```
